test/unit/lodash.spec.js

```diff
@@ -0,0 +1,583 @@
+                    'green': 2,
+                });
+            });
+        });
+    });
+
+    describe('pickBy', function() {
+        describe('pickBy1', function() {
+            beforeEach(function() {
+                params = datasets.pickBy1();
+            });
+            it('should match types of passing elements', function() {
+                var element = params[0] instanceof Object;
+                var element1 = params[1] instanceof Function;
+                expect(element).to.eql(true);
+                expect(element1).to.eql(true);
+                expect(params.length).to.eql(2);
+            });
+            it('should pick ', function() {
+                expect(_.pickBy.apply(_, params)).to.eql({
+                    'x': 16,
+                });
+            });
+        });
+        describe('pickBy2', function() {
+            beforeEach(function() {
+                params = datasets.pickBy2();
+            });
+            it('should match types of passing elements', function() {
+                var element = params[0] instanceof Object;
+                var element1 = params[1] instanceof Function;
+                expect(element).to.eql(true);
+                expect(element1).to.eql(true);
+                expect(params.length).to.eql(2);
+            });
+            it('should pick ', function() {
+                expect(_.pickBy.apply(_, params)).to.eql({
+                    'x': 16,
+                });
+            });
+        });
+        describe('pickBy3', function() {
+            beforeEach(function() {
+                params = datasets.pickBy3();
+            });
+            it('should match types of passing elements', function() {
+                var element = params[0] instanceof Object;
+                var element1 = params[1] instanceof Function;
+                expect(element).to.eql(true);
+                expect(element1).to.eql(true);
+                expect(params.length).to.eql(2);
+            });
+            it('should pick ', function() {
+                expect(_.pickBy.apply(_, params)).to.eql({
+                    'x': 16,
+                });
+            });
+        });
+        describe('pickBy4', function() {
+            beforeEach(function() {
+                params = datasets.pickBy4();
+            });
+            it('should match types of passing elements', function() {
+                var element = params[0] instanceof Object;
+                var element1 = params[1] instanceof Function;
+                expect(element).to.eql(true);
+                expect(element1).to.eql(true);
+                expect(params.length).to.eql(2);
+            });
+            it('should pick ', function() {
+                expect(_.pickBy.apply(_, params)).to.eql({
+                    'x': 16,
+                });
+            });
+        });
+    });
+
+    describe('set', function() {
+        describe('set1', function() {
```

Comment:
For that exercise it would be definitely better to provide/pass to execution fixed object and just check its properties instead of trying to enforce user to create object itself. I can easily bypass the tests as they are right now.